.travis.yml

```diff
@@ -103,26 +103,31 @@ jobs:
     - git clone -b latest https://github.com/RasaHQ/starter-pack-rasa-stack.git
     - cd starter-pack-rasa-stack
     - python -m pytest tests
-  - stage: integration
-    name: "Test Docs Core"
-    install:
-    - pip install -r requirements-docs.txt
-    - pip install -e .[sql]
-    - pip list
-    script:
-      # be strict about warnings --> they will be treated as errors
-    - cd docs/core
-    - make SPHINXOPTS="-W --keep-going -A html_theme=rasabaster" html
```

Comment:
@amn41 this shouldn't be removed, it checks the docs are sane (e.g. all the links exist)

Reply:
ok! will create a new PR bc I guess this will throw some errors